msal/src/main/java/com/microsoft/identity/client/IPublicClientApplication.java

```diff
@@ -111,4 +125,20 @@ void acquireToken(@NonNull final Activity activity,
      * @return
      */
     boolean isSharedDevice();
+
+    interface LoadAccountsCallback extends TaskCompletedCallbackWithError<List<IAccount>, MsalException> {
+        /**
+         * Called once succeed and pass the result object.
+         *
+         * @param result the success result.
+         */
+        void onTaskCompleted(List<IAccount> result);
+
+        /**
+         * Called once exception thrown.
+         *
+         * @param exception
+         */
+        void onError(MsalException exception);
```

Comment:
I am trying to compare the api of `LoadAccountsCallback`  with `CurrentAccountCallback`, which takes `Exception` class for onError call ```void onError(@NonNull final Exception exception);```.  Should it be consistency with the onError parameter's type for LoadAccountsCallback and CurrentAccountCallback? It could all take MsalException as para, where we can adapt the Exception into MsalException.

Reply:
I missed this. CurrentAccountCallback should also return MsalException.